packages/components/src/mobile-promo-card/style.scss

```diff
@@ -51,6 +51,12 @@
 
 .promo-card__icons {
 	margin-bottom: 24px;
+	max-height: 55px;
+
+	.woo-icon {
+		height: 55px;
```

Comment:
Since those two values are linked, you might want to use a variable.

Reply:
Oh, good call! I'll definitely add that.